client/my-sites/people/team-invite/index.tsx

```diff
@@ -56,7 +56,7 @@ function TeamInvite( props: Props ) {
 				<PageViewTracker path="/people/new/:site" title="People > Invite People" />
 				<HeaderCake onClick={ goBack }>
 					{ translate( 'Add team members to %(sitename)s', {
-						args: { sitename: site.name },
+						args: { sitename: site.name ?? '' },
```

Comment:
Perhaps we can change the string to default to "this site"?

Reply:
Added